CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -2992,6 +2992,7 @@ end
 
 local function getDeveloperConsole()
 	if not myDeveloperConsole then
+		myDeveloperConsole = true -- debounce
```

Comment:
I don't know how any of this new github ui works but this change will be a part of #540.